docs/src/modules/ROOT/pages/constraints-and-score/performance.adoc

```diff
@@ -4,35 +4,44 @@
 [#scoreCalculationPerformanceTricksOverview]
 == Overview
 
-The `Solver` will normally spend most of its execution time running score calculation,
+The `Solver` will normally spend most of its execution time evaluating moves and running score calculation,
 which is called in its deepest loops.
-Faster score calculation will return the same solution in less time with the same algorithm,
+Faster move evaluation will return the same solution in less time with the same algorithm,
```

Comment:
In this case, we are arguably talking about faster score calculation.
That is the deciding factor for speed, if everything else is equal. Move evaluation speed is _determined_ by score calculation speed.